test/src/test/groovy/hudson/cli/EnableJobCommandTest.groovy

```diff
@@ -42,8 +50,7 @@ public class EnableJobCommandTest extends HudsonTestCase {
             cli.execute(["enable-job",p.name])
             assertFalse(p.disabled)
         } finally {
-            cli.close();
+            cli.close()
```

Comment:
Shame on me. I always try to be as focused as possible because I know how hard it is to review tons of unrelated changes.